lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -191,69 +190,69 @@ <h3 class="course-title">
                     <li class="actions-item" id="actions-item-unenroll-${dashboard_index}">
                     % if is_paid_course and show_refund_option:
                       % if not is_course_blocked:
-                      <a href="#unenroll-modal" class="action action-unenroll" rel="leanModal" data-course-id="${course_overview.id | h}" data-course-number="${course_overview.number | h}" data-course-name="${course_overview.display_name_with_default_escaped | h}" data-dashboard-index="${dashboard_index}"
```

Comment:
Is the `display_name_with_default_escaped | h` a double-escaping bug (that you fixed)?  Did you verify that the fix didn't break anything?